nixos/modules/services/misc/plex.nix

```diff
@@ -30,6 +30,30 @@ in
         default = "plex";
         description = "Group under which Plex runs.";
       };
+
+
+      managePlugins = mkOption {
```

Comment:
The point is that it only removes the symlinks when `managePlugins` is set to true. Otherwise it just leaves them alone (similarly to how the imperative users option doesn't remove manually added users).